### ISSUE TITLE
[a11y]: Tooltip is with violation "Content is not within a landmark element"

### ISSUE THREAD

**Issue by PaigeChen** [carbon-design-system/carbon#9893]
### Package

carbon-components-react

### Browser

Chrome

### Operating System

MacOS

### Package version

7.46 (latest)

### Automated testing tool and ruleset

IBM Accessibility Equal Access Toolkit: Accessibility Checker 27 August 2021 - Latest Deployment

### Description

Run IBM A11y checker against the the carbon tooltip element with tooltip opened. 2 violations are reported, 1 of them, Content is not within a landmark element, is very similar to the closed issue https://github.com/carbon-design-system/carbon/issues/6308.
![image](https://user-images.githubusercontent.com/54969794/137872352-d26703cf-5eb8-4259-ad6b-ed6bbcb4949e.png)

### CodeSandbox example

https://fmtrj.csb.app/  (Directly generate the sample from Live Demo page)

### Steps to reproduce

1. Open https://www.carbondesignsystem.com/components/tooltip/usage/
2. Click "CodeSandbox" in Live Demo section
3. Click the "i" icon to open tooltip
4. Run IBM A11y checker

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by PaigeChen** [comment]
Hi @dakahn , would you mind to advise how can I reopen this ticket and how to fix this in our product's code? We met this issue our product first, but couldn't solve it. Please find details in https://github.com/carbon-design-system/carbon/issues/9893#issuecomment-953668621. Thanks.
